src/openforms/utils/migrations_utils/fix_default_translation.py

```diff
@@ -0,0 +1,42 @@
+import functools
```

Comment:
Are we going to keep this solution?I thought it was not the what we wanted.(asking since this is now a reusable thing..)

Reply:
it's not what we want, but it is what we need for the time being :grimacing: